src/MyEventEmitter.js

```diff
@@ -1,14 +1,79 @@
 'use strict';
 
 class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  constructor() {
+    this.events = {};
+  }
+
+  on(eventName, listener) {
+    if (!this.events[eventName]) {
+      this.events[eventName] = [];
+    }
+
+    this.events[eventName].push(listener);
+  }
+
+  once(eventName, listener) {
+    const deleteItself = (...args) => {
+      listener(...args);
+
+      this.off(eventName, deleteItself);
+    };
+
+    this.on(eventName, deleteItself);
+  }
+
+  off(eventName, listener) {
+    if (!this.events[eventName]) {
+      return;
+    }
+
+    this.events[eventName] = this.events[eventName].filter(
+      (l) => l !== listener,
+    );
```

Comment:
The `off` method uses strict equality (`!==`) to compare listeners. This may not work as expected if the same listener function is wrapped in different closures, such as when using the `once` or `prependOnceListener` methods. Consider using a mechanism to track and compare the original listener functions to ensure they can be removed correctly.